mcp_server.py

```
@@ -280,7 +357,17 @@ def create_mcp_server():
-    logger.info(f"MCP server created with 7 HTTP client tools")
+    server.tool(
+        name="add_device", 
+        description="Add a new device to the infrastructure registry"
+    )(add_device)
+    
+    server.tool(
+        name="remove_device",
+        description="Remove a device from the infrastructure registry"
+    )(remove_device)
+    
+    logger.info(f"MCP server created with 9 HTTP client tools")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)